devices/stm32f100.yaml

```diff
@@ -32,6 +32,7 @@ _include:
  - ../peripherals/pwr/pwr_v1.yaml
  - ../peripherals/gpio/gpio_f1.yaml
  - ../peripherals/spi/spi_v1_without_FRF_FRE_UDR_CHSIDE.yaml
+ - common_patches/tim/tim13_missing_opm.yaml
```

Comment:
```suggestion
```

In RM0009 TIM10/11/13/14 doesn't have OPM, I guess TIM14 is derived from TIM13 here because it's getting it too?

Reply:
RM0041 Rev 5 has OPM bits for TIM13/14